### PR TITLE
change kubeadm kind to test HA

### PR DESCRIPTION
**Pull request by RA489**
change kind-master job (and possibly kind-1.14) to test HA

refs https://github.com/kubernetes/kubeadm/issues/1567

### CODE REVIEW

**Comment by RA489** [comment]
depend upon https://github.com/kubernetes/kubeadm/pull/1571

File: config/jobs/kubernetes-security/generated-security-jobs.yaml [hunk 1]
```diff
@@ -246,7 +246,7 @@ presubmits:
         - --provider=skeleton
         - --deployment=kind
         - --kind-binary-version=build
-        - --kind-config-path=./../../k8s.io/kubeadm/tests/e2e/kind/single-cp/single-cp.yaml
+        - --kind-config-path=./../../k8s.io/kubeadm/tests/e2e/kind/ha-cp/ha-cp.yaml
         - --build=bazel
```
Comment (fabriziopandini) [inline config/jobs/kubernetes-security/generated-security-jobs.yaml]: I'm not sure we should change kubernetes-security/generated-security-jobs.yaml jobs.
I don't know who owns them and what they are meant to
@neolit123 have you got some context here?

File: config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml [hunk 2]
```diff
@@ -41,7 +41,7 @@ periodics:
       - --provider=skeleton
       - --deployment=kind
       - --kind-binary-version=build
-      - --kind-config-path=./../../k8s.io/kubeadm/tests/e2e/kind/single-cp/single-cp.yaml
```
Comment (fabriziopandini) [inline config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml]: [1] only v1.13 and v1.14 jobs should be changed

File: config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml [hunk 3]
```diff
@@ -120,7 +120,7 @@ periodics:
...
+      - --kind-config-path=./../../k8s.io/kubeadm/tests/e2e/kind/ha-cp/ha-cp.yaml
```
Comment (fabriziopandini) [inline config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml]: [1]

File: config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml [hunk 4]
```diff
@@ -429,7 +429,7 @@ presubmits:
         - --provider=skeleton
         - --deployment=kind
         - --kind-binary-version=build
-        - --kind-config-path=./../../k8s.io/kubeadm/tests/e2e/kind/single-cp/single-cp.yaml
+        - --kind-config-path=./../../k8s.io/kubeadm/tests/e2e/kind/ha-cp/ha-cp.yaml
```
Comment (fabriziopandini) [inline config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml]: [1]

File: config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml [hunk 5]
```diff
@@ -355,7 +355,7 @@ presubmits:
...
```
Comment (fabriziopandini) [inline config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml]: [1]

File: config/jobs/kubernetes-security/generated-security-jobs.yaml (hunk 1 above)
Comment (neolit123) [reply config/jobs/kubernetes-security/generated-security-jobs.yaml]: the security jobs are auto-generated via ./hack/update-config.sh
if the script is not updating them we should not touch them.

@RA489 
is this change the result of the script?

**Comment by neolit123** [comment]
> The change should apply only to 1.14 and master because HA in the older releases has different behavior.

+1

please only update 1.14 and master. forgot to mention that...

File: config/jobs/kubernetes-security/generated-security-jobs.yaml (hunk 1 above)
Comment (RA489) [reply config/jobs/kubernetes-security/generated-security-jobs.yaml]: @neolit123 ./hack/update-config.sh script updated the security jobs.

**Comment by neolit123** [comment]
/assign @krzyzacy 
PTAL in terms of the security job update (auto-generated).

**Comment by neolit123** [comment]
/assign @spiffxp 
(not sure if @krzyzacy is on PTO)

**Comment by spiffxp** [comment]
/approve
So I'm wary of approving changes to the job all over the place during code freeze when it's used for CI signal.  Let's see how this looks and rollback if necessary.

FYI @alejandrox1 @jimangel @smourapina @alenkacz

**Pull request merged**